packages/botonic-react/src/components/message.jsx

```diff
@@ -92,52 +117,75 @@ export const Message = props => {
     return (
       <div
         style={{
+          display: 'flex',
+          justifyContent: isFromUser() ? 'flex-end' : 'flex-start',
           position: 'relative',
-          alignSelf: isFromUser() ? 'flex-end' : 'flex-start',
-          margin: 8,
-          backgroundColor: getBgColor(),
-          color: isFromUser() ? '#fff' : '#000',
-          border: `1px solid ${getBgColor()}`,
-          fontFamily: 'Arial, Helvetica, sans-serif',
-          borderRadius: 8,
-          maxWidth: blob ? '60%' : 'calc(100% - 16px)',
-          ...style,
+          paddingLeft: 5
         }}
-        {...otherProps}
       >
-        <div
+        <div style={{ width: 30, position: 'absolute', bottom: 0 }}>
+          {isFromBot() && webchatState.theme.botLogoChat ? (
+            <webchatState.theme.botLogoChat />
+          ) : (
+            isFromBot() && <img style={{ width: 30 }} src={Logo} />
+          )}
+        </div>
+
+        <DefaultMessage
           style={{
-            padding: '8px 12px',
-            display: 'flex',
-            flexDirection: 'column',
-            whiteSpace: 'pre-line'
+            left: isFromBot() ? 30 : 0,
+            top: isFromBot() ? 5 : 0,
+            backgroundColor: getBgColor(),
+            color:
+              webchatState.theme.customUserMessages ||
```

Comment:
since the color calculation is complex, maybe we can set it in a variable before the return at line 117?